include/triton/core/tritonbackend.h

```diff
@@ -978,7 +978,10 @@ TRITONBACKEND_ModelAutoCompleteConfig(
 /// scheduling choice can only be changed if none is previously set. Any other
 /// changes to the model configuration will be ignored by Triton. This function
 /// can only be called from TRITONBACKEND_ModelInitialize, calling in any other
-/// context will result in an error being returned. The function does not take
+/// context will result in an error being returned. Additionally, Triton server
+/// can add some of the missing fields in the provided config with this call.
```

Comment:
Is this the new behavior or it existed before too and we are just trying to better document it?

Reply:
It is a new behavior added by enabling dynamic batching as part of auto-complete.